api/src/pcapi/local_providers/provider_manager.py

```diff
@@ -85,35 +89,52 @@ def synchronize_venue_provider(venue_provider: VenueProvider, limit: int | None
         )
 
 
-def synchronize_ems_venue_providers() -> None:
-    cinemas_programs = ems.get_cinemas_programs()
-    version = cinemas_programs.version
+def synchronize_ems_venue_providers(from_last_version: bool = False) -> None:
+    last_version = 0
+    venue_provider_to_sync: set[int] = set()
+    venue_provider_by_site_id: dict[str, VenueProvider] = {}
+
+    if from_last_version is True:
+        last_version = get_ems_oldest_sync_version()
```

Comment:
(détail / style) Plus courant (et concis) :

```suggestion
    if from_last_version is True:
        last_version = get_ems_oldest_sync_version()
```

Reply:
je ne vois pas la différence avec ce que j'ai fais ^^"

Reply:
C'est parce que je ne suis pas doué avec tous ces claviers, souris, etc. Je le refais, en m'appliquant cette fois : 

```suggestion
    if from_last_version:
        last_version = get_ems_oldest_sync_version()
```

;)